src/pyseekdb/client/client_base.py

```
@@ -101,16 +101,18 @@ def _validate_collection_name(name: str) -> None:
         ValueError: If name is empty, too long, or contains invalid characters.
     """
     if not isinstance(name, str):
-        raise TypeError(f"Collection name must be a string, got {type(name).__name__}")
+        raise TypeError(
+            f"Invalid collection name: '{name}'. Collection name must be a string, got {type(name).__name__}"
+        )
```

Ruff (0.14.14)

Avoid specifying long messages outside the exception class

(TRY003)